war/src/main/js/add-item.js

```diff
@@ -1,6 +1,12 @@
 // Initialize all modules by requiring them. Also makes sure they get bundled (see gulpfile.js).
 var $ = require('jquery-detached').getJQuery();
 
+// Minimum number of item by category to draw the category
+var MIN_ITEM = 6;
```

Comment:
This is a pretty big number IMHO. There are not so many Item types in Jenkins, so would be useful to know reason of such value